rero_ils/modules/items/api/record.py

```diff
@@ -475,7 +474,9 @@ def holding_library_pid(self):
     @property
     def organisation_pid(self):
         """Get organisation pid for item."""
-        library = Library.get_record_by_pid(self.library_pid)
+        if self.get('organisation'):
+            return extracted_data_from_ref(self.get('organisation'))
+        library = self.get_library()
         return library.organisation_pid
```

Comment:
COSMETIC : could be merge to save memory allocation : `return self.get_library().organisation_pid`